### DISCUSSION TITLE
Why!!!

### DISCUSSION THREAD

**Discussion by Green-Pepper-Bibimbap** [community/community#131816] [GitHub Education]
### Hi everyone,

Why it always tell me that "Please ensure that your academic affiliation document contains your last name exactly as it appears in your GitHub billing information. If you have only a single legal name, enter it in both the first and last name fields of your GitHub billing information."
Every time.
![image](https://github.com/community/community/assets/162942287/ea5a9f30-c8b3-4309-a2bf-b62888582880)

**Comment by naveenalla3000** [comment]
please ensure that the name on your academic affiliation document (e.g., student ID, enrollment verification) matches your legal name. Go to GitHub's billing settings and ensure your last name in the billing information matches exactly how it appears on your academic document. If you have only a single legal name, enter it in both the first and last name fields.After making the necessary changes, reupload your academic affiliation document to GitHub.

mark me correct answer if it help you.

**Comment by samus-aran** [comment]
Hi @Green-Pepper-Bibimbap 👋 

I suspect the style of characters may be impacting your application. If you have manually translated your document try and use Capital characters in case certain characters are being misrecognised. 

Let us know how you get on. 👍
